src/common/styles/icons.m.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+	font-family: 'dojo2';
+	src:
+		url('fonts/dojo2.ttf?k76fx7') format('truetype'),
+		url('fonts/dojo2.woff?k76fx7') format('woff'),
+		url('fonts/dojo2.svg?k76fx7#dojo2') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+.icon {
+	speak: none;
+	font-style: normal;
+	font-weight: normal;
+	font-variant: normal;
+	text-transform: none;
+	line-height: 1;
+	display: inline-block;
+	font: normal normal normal 14px/1 'dojo2';
+	font-size: inherit;
+	text-rendering: auto;
+
+	/* Better Font Rendering =========== */
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
+.plus:before {
+	content: "\f067";
+}
+.minus:before {
+	content: "\f068";
+}
+.check:before {
+	content: "\f00c";
+}
+.close:before {
```

Comment:
How does this work with themes? It would be nice to show different icons when the theme check box is toggled. I feel like that would really cover all aspects of theming.